### PR TITLE
Update links in `/links` command

### PR DESCRIPTION
**Pull request by soupless**
Updated YouTube link to use mCoding's handle, and added the [mCoding site](https://mcoding.io).

### CODE REVIEW

**Review by CircuitSacul — APPROVED** [review]

File: mcodingbot/plugins/info.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ async def links(ctx: Context) -> None:
         title="Links",
         color=CONFIG.theme,
         description=(
+            f"**[mCoding site]({CONFIG.mcoding_site})**\n"
```
Comment (Lunarmagpie) [inline mcodingbot/plugins/info.py:23]: Maybe?
```suggestion
            f"**[mCoding website]({CONFIG.mcoding_site})**\n"
```

Comment (trag1c) [reply mcodingbot/plugins/info.py:23]: Also rename `mcoding_site` to `mcoding_website` for consistency

File: mcodingbot/plugins/info.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ async def links(ctx: Context) -> None:
         title="Links",
         color=CONFIG.theme,
         description=(
-            f"**[mCoding site]({CONFIG.mcoding_site})**\n"
+            f"**[mCoding website]({CONFIG.mcoding_website})**\n"
             f"**[mCoding Youtube]({CONFIG.mcoding_youtube})**\n"
```
Comment (trag1c) [inline mcodingbot/plugins/info.py:24]:
```suggestion
            f"**[mCoding YouTube]({CONFIG.mcoding_youtube})**\n"
```
i guess it's not out of scope to suggest that

Comment (Lunarmagpie) [reply mcodingbot/plugins/info.py]: agreed

**Review by Lunarmagpie — APPROVED** [review]

**Review by trag1c — APPROVED** [review]

**Pull request merged**